### PR TITLE
Update Burmese, Japanese and four other navs to remove deprecated indexes

### LINKED ISSUE #8418: NAV: Update the nav for Burmese, Japanese, Pashto, Somali, Thau and Vietnamese to remove election and deprecated indexes
**Is your feature request related to a problem? Please describe.**
Update the nav for Update the nav for Burmese, Japanese, Pashto, Somali, Thai and Vietnamese

**Describe the solution you'd like**
Make the following changes: 

JAPANESE
Update: 
```     {
        title: 'ホーム',
        url: '/japanese',
      },
      {
        title: 'コロナウイルス',
        url: '/japanese/52137815',
      },
      {
        title: '米大統領選',
        url: '/japanese/53969845',
      },
      {
        title: '日本',
        url: '/japanese/topics/cyx5k201n3qt',
      },
      {
        title: 'アジア',
        url: '/japanese/topics/cyx5k20kvd2t',
      },
      {
        title: 'イギリス',
        url: '/japanese/topics/c95y3gk44nyt',
      },
      {
        title: 'アメリカ',
        url: '/japanese/topics/cdr56kqdr70t',
      },
      {
        title: '解説・読み物',
        url: '/japanese/features-and-analysis-54539120',
      },
      {
        title: 'ビデオ',
        url: 'https://www.bbc.com/japanese/video',
      },
      {
        title: 'ワールドニュースTV',
        url: 'https://www.bbcworldnews-japan.com/',
      },
```
BURMESE
Update navigation:

 ```
{
        title: 'ပင်မစာမျက်နှာ',
        url: '/burmese',
      },
      {
        title: 'မြန်မာ့ရေးရာ',
        url: '/burmese/topics/c404v08p1wxt',
      },
      {
        title: 'နိုင်ငံတကာ',
        url: '/burmese/topics/cnlv9j1z93wt',
      },
      {
        title: 'ဆောင်းပါး',
        url: '/burmese/in-depth-54539126',
      },
      {
        title: 'အားကစား',
        url: '/burmese/topics/cxnykndgd87t',
      },
      {
        title: 'ကုန်သွယ်စီးပွား',
        url: '/burmese/topics/c9wpm0en9jdt',
      },
      {
        title: 'ဗီဒီယိုများ',
        url: '/burmese/media/video',
      },
```

PASHTO

Replace
 ```
     {
        title: 'ځانګړې پاڼې',
        url: '/pashto/in_depth',
      },
```

With

```
      {
        title: 'ځانګړې پاڼې',
        url: '/pashto/in-depth-54540873',
      },
```
SOMALI
Update

```
      {
        title: 'War',
        url: '/somali',
      },
      {
        title: 'Ganacsi',
        url: '/somali/topics/c2dwqd32v4yt',
      },
      {
        title: 'Cayaaraha',
        url: '/somali/topics/cpzd4zj1pn2t',
      },
      {
        title: 'Muuqaal',
        url: '/somali/media/video',
      },
      {
        title: 'Barnaamijyada Idaacadda',
        url: '/somali/media-54071665',
      },

```

THAI
Remove:
```
      {
        title: 'เลือกตั้งสหรัฐฯ 2020',
        url: '/thai/international-54245234',
      },
```

VIETNAMESE
Update nav:
```
      {
        title: 'Tin chính',
        url: '/vietnamese',
      },
      {
        title: 'Việt Nam',
        url: '/vietnamese/topics/ckdxnx1x5rnt',
      },
      {
        title: 'Thế giới',
        url: '/vietnamese/topics/cnlv9j1ekq0t',
      },
      {
        title: 'Diễn đàn',
        url: '/vietnamese/-54540875',
      },
      {
        title: 'Kinh tế',
        url: '/vietnamese/topics/cez1ey7nzj3t',
      },
      {
        title: 'Nhịp sống mới',
        url: '/vietnamese/magazine-54029179',
      },
      {
        title: 'Thể thao',
        url: '/vietnamese/topics/ckdxnx1k7zxt',
      },
      {
        title: 'Video',
        url: '/vietnamese/media/video',
      },

```

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by jordit**
Resolves #8418 

**Overall change:**
_A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- Replaces links to recent assets with links to topic pages on BBC Burmese Nav 
- Updated Features link on BBC Japanese Nav
- Updated In depth link on BBC Pashto Nav
- Removed links to News and Knowledge recent assets index on BBC Somali Nav
- Removed US election link from BBC Thai nav
- Updated links on BBC Vietnamese nav and removed Learning English 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/vietnamese.js [hunk 1]
```diff
@@ -249,31 +249,27 @@ export const service = {
...
       },
       {
         title: 'Thế giới',
-        url: '/vietnamese/world',
+        url: '/vietnamese/topics/cnlv9j1ekq0t',
       },
       {
         title: 'Diễn đàn',
-        url: '/vietnamese/forum',
+        url: '/vietnamese/-54540875',
```
Comment (jimjohnsonrollings) [inline src/app/lib/config/services/vietnamese.js:260]: @jordit This one doesn't look right - is it missing something, or does it need the dash taken out

**Comment by jimjohnsonrollings** [comment]
Looks like the snapshots for integration tests need updating - 28 failures on CI.

**Review by jimjohnsonrollings — CHANGES_REQUESTED** [review]

File: src/app/lib/config/services/vietnamese.js (hunk 1 above)
Comment (sareh) [inline src/app/lib/config/services/vietnamese.js:260]:
```suggestion
        url: '/vietnamese/forum-54540875',
```

**Comment by andrewscfc** [comment]
Moved to 'issue in progress' while the outstanding issues are addressed

File: src/app/lib/config/services/vietnamese.js (hunk 1 above)
Comment (joshcoventry) [reply src/app/lib/config/services/vietnamese.js:260]: @jordit Is the correct link `/vietnamese/forum-54540875` ?

**Comment by jordit** [comment]
Superseded

**Pull request closed without merging**